app/ante/cosmos/min_price.go

```diff
@@ -35,10 +39,19 @@ func (mpd MinGasPriceDecorator) AnteHandle(ctx sdk.Context, tx sdk.Tx, simulate
 
 	minGasPrice := mpd.feesKeeper.GetParams(ctx).MinGasPrice
 
+	feeCoins := feeTx.GetFee()
+
+	// only allow user to pass in aevmos native token as transaction fees
+	validFees := len(feeCoins) == 0 || (len(feeCoins) == 1 && feeCoins.GetDenomByIndex(0) == config.BaseDenom)
```

Comment:
`config.BaseDenom` is hardcoded, please use the EVM denom instead (line 56)

Reply:
done